vscode-extensions/rush-vscode-extension/package.json

```diff
@@ -277,7 +277,7 @@
     "glob": "~7.0.5",
     "local-node-rig": "workspace:*",
     "mocha": "^10.1.0",
-    "vsce": "~2.14.0"
+    "@vscode/vsce": "3.3.2"
```

Comment:
Consume in plugin directly as a dependency (see other comment on vsix heft plugin